src/main/kotlin/com/terraformation/backend/accelerator/db/DeliverablesImporter.kt

```diff
@@ -73,6 +74,16 @@ class DeliverablesImporter(
         }
 
         val deliverableId = values[COLUMN_ID]?.toLongOrNull()?.let { DeliverableId(it) }
+        val deliverableType =
+            values[DELIVERABLE_TYPE]?.lowercase()?.let {
+              when (it) {
+                "document" -> DeliverableType.Document
+                "species" -> DeliverableType.Species
+                else ->
+                    throw IllegalArgumentException(
+                        "Invalid Document Type on row $rowNumber - $it, must be one of: 'Document', 'Species'")
+              }
+            }
```

Comment:
Is it valid to leave the field blank so this ends up being null? I would think we'd want to make the deliverable type mandatory, in which case following a pattern like we do for category might be better, where there's a map of lowercase names to enums and this becomes something like
```kotlin
val deliverableType = values[DELIVERABLE_TYPE]?.lowerCase()?.let { deliverableTypesByName[it] }
    ?: throw IllegalArgumentException("same message you have in the PR")
```

Reply:
I ended up following the pre-existing pattern for validating as seen with the category column